stdlib/arg.ml

```diff
@@ -264,6 +263,14 @@ let parse_and_expand_argv_dynamic_aux allow_expand current argv speclist anonfun
     end;
   done
 
+let parse_and_expand_argv_dynamic_aux allow_expand current argv speclist anonfun
```

Comment:
Is this a shadowing definition? There are some many `parse_` functions now that I think it would be best avoided for readability.